modules/google/etcd/variables.tf

```diff
@@ -14,6 +14,14 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+variable "external_endpoints" {
+  type = "list"
+}
+
+variable "instance_count" {
+  default = "3"
```

Comment:
we try to avoid defaults in modules as a best practice, in order to enforce setting values in the invoking platforms module.